requirements.txt

```diff
@@ -0,0 +1 @@
+pycairo==1.24.0
```

Comment:
Can you let me know which operating system / environment you tried this in? I haven't tried updating pycairo for a while, I remember I tried a couple years ago and ran into issues. Probably time for me to try again.

Reply:
ubuntu 22.04
python 3.10.6

There is no special reason for choosing this version -- simply it is the current one and it works